### ISSUE TITLE
[py2py3] pre-production test of MSOutput with the py3 environment

### ISSUE THREAD

**Issue by mapellidario** [dmwm/WMCore#10429]
**Impact of the new feature**

This should involve mainly MSOutput and its dependencies.

**Is your feature request related to a problem? Please describe.**

After making sure that the whole WMCore passes unit tests in py3, we should run some pre-productions tests to check if MSOutput is compatible with py3 in a production-like environment.

**Describe the solution you'd like**

We should deploy MSOutput to testbed, run some workflows and see how it behaves. (please feel free to correct me or expand this section of the issue with more details!)

**Describe alternatives you've considered**
There is not much else that we can do. 

**Additional context**
I would tackle this after we have completed 
- [ ] #10422

**Comment by amaltaro** [comment]
This one is slightly trick to validate in testbed or in our VM, unless we point it (and the agent feed data into Rucio) to use Rucio Production server. Problem with the Int one is that it fails to resolve the TAPE rse expression, which means, it never manages to get a workflow completely done in the cycle.
It also made me realize that we should make whatever fixes we need (either on our side, or on Rucio Int) to make sure that our pre-production system is as close as the production one, thus providing us a high level of confidence when testing things before pushing them to production.

Short version, Python3 MSOutput version `1.5.0.pre6` seems to be running okay, at least the Disk part. Tomorrow we will be able to confirm it and hopefully close this ticket.

**Comment by amaltaro** [comment]
I can confirm that Python3 MSOutput is fully functional in CMSWEB production. I've seen both Disk and Tape rules being made and etc.
Closing this ticket.
